frontend/src/static/js/main.js

```diff
@@ -250,7 +250,11 @@ window.app = new window.Vue({
       return JSON.stringify(dataObj);
     },
 
-    getRepoSenseLink() {
+    getRepoSenseHomeLink() {
+      return 'reposense.org';
```

Comment:
It has to contain the protocol i.e. `http://reposense.org` else its being appended as a path.

Reply:
You can try it on [Netlify](https://deploy-preview-883--reposense.netlify.com/).